packages/design-system/modal/modal.container.web.tsx

```diff
@@ -22,7 +24,7 @@ const MODAL_CONTAINER_TW = [
   "bg-white dark:bg-black",
   "shadow-xl shadow-black dark:shadow-white",
   "rounded-t-[32px] rounded-b-0 sm:rounded-b-[32px] pb-4",
-  "max-h-100vh",
+  "max-h-70vh sm:max-h-90vh",
```

Comment:
just tips, not blockers, last time we discussed using `max-h-82vh` on the desktop.
and could you publish this component on npm! thank you! 🙏